test/Swashbuckle.AspNetCore.TestSupport/Fixtures/TypeWithNullableContext.cs

```diff
@@ -3,8 +3,41 @@
 namespace Swashbuckle.AspNetCore.TestSupport
 {
 #nullable enable
-    public class TypeWithNullableContext
+
+    // These types are used to test our handling of nullable references types.
+    // NRT results in NullableContextAttribute and NullableAttribute being placed on
+    // types and members by the compiler.
+
+    // Remember to mirror both types and use both types in tests.
+
+    /// <summary>
+    /// We expect this type to receive NullableContext(1) (NotAnnotated) from the compiler.
+    /// </summary>
+    public class TypeWithNullableContextNotAnnotated
     {
+        // Dummies to affect the NullableContextAttribute value.
+        // It seems to default to the most common nullable state.
+        public string Dummy1 { get; } = default!;
+        public string Dummy2 { get; } = default!;
+        public string Dummy3 { get; } = default!;
+        public string Dummy4 { get; } = default!;
+        public string Dummy5 { get; } = default!;
+        public string Dummy6 { get; } = default!;
+        public string Dummy7 { get; } = default!;
+        public string Dummy8 { get; } = default!;
+        public string Dummy9 { get; } = default!;
+        public string Dummy10 { get; } = default!;
+        public string Dummy11 { get; set; } = default!;
+        public string Dummy12 { get; set; } = default!;
+        public string Dummy13 { get; set; } = default!;
+        public string Dummy14 { get; set; } = default!;
+        public string Dummy15 { get; set; } = default!;
+        public string Dummy16 { get; set; } = default!;
+        public string Dummy17 { get; set; } = default!;
+        public string Dummy18 { get; set; } = default!;
+        public string Dummy19 { get; set; } = default!;
+        public string Dummy20 { get; set; } = default!;
```

Comment:
Why so many properties?

Reply:
I didn't find any other way of coercing the compiler to change the flag in `NullableContextAttribute`. The context flag seems to pick the most common case as an optimization, so that as many members as possible can avoid having `NullableAttribute` at all. But our logic uses the two attributes to figure things out, so the tests need to be able to replicate both nullable and non-nullable cases.

So:
- Many non-nullable -> `NullableContext(1)`
- Many nullable -> `NullableContext(2)`

When we remove all the manual logic related to `Nullable` and `NullableContext` at some point, we can probably simplify and remove a lot of this awkward compiler coercion, and just trust that Microsoft's `NullabilityInfoContext` does a good job.

Reply:
Thanks for explaining.

Reply:
This is interesting. So it's the large number of properties that "tips" the compiler over into doing the "optimized" format? Is the optimized format described somewhere in the Roslyn docs or somewhere else? Thank you for the improved fix!